### PR TITLE
Merge all code from FO Core

### PR DESCRIPTION
**Pull request by autotelik**
Remove usage of FO Core gem
Merge code from core gem into this app

### CODE REVIEW

File: bin/rails [hunk 1]
```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path("../spring", __FILE__)
-rescue LoadError
-end
```
Comment (TThurston) [inline bin/rails]: I found that these changes weren't necessary.  Can we confirm if they are needed, or just a temporary change during trouble-shooting?

File: bin/rake [hunk 2]
```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path("../spring", __FILE__)
-rescue LoadError
-end
```
Comment (TThurston) [inline bin/rake]: I found that these changes weren't necessary.  Can we confirm if they are needed, or just a temporary change during trouble-shooting?

File: bin/rails (hunk 1 above)
Comment (autotelik) [inline bin/rails]: hmm odd ... just took a fresh clone, on a different PC, and even on develop branch I get 

[14:59:15:rubyuser:~/SoftwareDev/git/waste-exemptions] >sr
bin/rails:6: warning: already initialized constant APP_PATH
/home/rubyuser/SoftwareDev/git/waste-exemptions/bin/rails:6: warning: previous definition of APP_PATH was here

So maybe an rvm / ruby-2.2.3 issue

**Comment by autotelik** [comment]
Looks good to me :ship:

**Pull request merged**